### PR TITLE
Release/0.8.0

### PR DESCRIPTION
**Pull request by oguzhanunlu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ limitations under the License.
 [travis]: https://travis-ci.org/snowplow/snowplow-mini
 [travis-image]: https://travis-ci.org/snowplow/snowplow-mini.svg?branch=master
 
-[release-image]: http://img.shields.io/badge/release-0.7.0-blue.svg?style=flat
+[release-image]: http://img.shields.io/badge/release-1.0.0-blue.svg?style=flat
```
Comment (benjben) [inline README.md]: Is it 0.8 or 1.0 ?

File: provisioning/resources/ui/js/components/Overview.tsx [hunk 2]
```diff
@@ -48,10 +48,10 @@ export class Overview extends React.Component<{}, {}> {
...
         <h3>The software stack installed: </h3>
         <ul>
-          <li>Snowplow Stream Collector NSQ 0.13.0</li>
-          <li>Snowplow Stream Enrich NSQ 0.21.0</li>
-          <li>Snowplow Elasticsearch Loader 0.10.1</li>
-          <li>Snowplow Iglu Server 0.3.0</li>
+          <li>Snowplow Stream Collector NSQ 1.0.0</li>
+          <li>Snowplow Stream Enrich NSQ 1.0.0</li>
+          <li>Snowplow Elasticsearch Loader 0.12.0</li>
+          <li>Snowplow Iglu Server 0.6.0</li>
```
Comment (paulboocock) [inline provisioning/resources/ui/js/components/Overview.tsx]: Now bumped to 0.6.1

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,12 @@
+Version 1.0.0 (2020-02-26)
+--------------------------
+Bump Stream Enrich to 1.0.0 (#221)
+Bump Elasticsearch Loader to 0.12.0 (#220)
+Fix postgres not starting after image update (#219)
+Configure collector to return SameSite=None; Secure cookie (#218)
+Bump Scala Stream Collector to 1.0.0 (#206)
+Bump Iglu Server to 0.6.0 (#205)
```
Comment (paulboocock) [inline CHANGELOG]: Now bumped to 0.6.1

File: README.md (hunk 1 above)
Comment (oguzhanunlu) [reply README.md]: nice catch, thanks @benjben ! it is 0.8.0, I'll fix it

**Pull request merged**